src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,31 @@
+package core.basesyntax;
+
+public class Ball {
+    private String color;
+    private int number;
+
+    public Ball(String color, int number) {
+        setColor(color);
+        setNumber(number);
+    }
```

Comment:
```suggestion
    public Ball(String color, int number) {
        this.color = color;
        this.number = number;
    }
```

let do not use setters in constructor: https://softwareengineering.stackexchange.com/questions/329806/why-did-it-not-become-a-common-pattern-to-use-setters-in-the-constructor https://stackoverflow.com/questions/4893558/calling-setters-from-a-constructor